config/sync/dpl_library_agency.general_settings.yml

```diff
@@ -0,0 +1,3 @@
+library_agency_name: null
+sms_notifications_for_patrons_enabled: 1
+reservation_sms_notifications_disabled: 0
```

Comment:
Please reexport the configuration or remove the `sms_notifications_for_patrons_enabled` and `library_agency_name` entries.

As far as I can tell the only proper entry is `reservation_sms_notifications_disabled`.

Reply:
Please remember to add all this configuration to our `config_ignore` configuration so local administrators can manage it freely.

Reply:
Yes, had in mind to get that part before I finished the PR